components/CheckDocumentation/vacuum/TxidWraparound.tsx

```diff
@@ -0,0 +1,110 @@
+import React from "react";
+
+import {
+  CheckDocs,
+  CheckGuidanceProps,
+  CheckTriggerProps,
+} from "../../../util/checks";
+import { useSmartAnchor } from "../../SmartAnchor";
+
+const TxidWraparoundTrigger: React.FunctionComponent<CheckTriggerProps> = ({
+  config,
+}) => {
+  const warning = config.settings["warning_threshold_pct"] as number;
+  const critical = config.settings["critical_threshold_pct"] as number;
+  return (
+    <p>
+      Detects when transaction ID space utilization on the server is more than{" "}
+      <code>{warning}%</code> and creates an issue with severity "warning".
+      Escalates to "critical" once utilization exceeds <code>{critical}%</code>.
+      Resolves once utilization falls below the threshold.
+    </p>
+  );
+};
+
+const TxidWraparoundGuidance: React.FunctionComponent<CheckGuidanceProps> = ({
+  urls: { backendsUrl, serverVacuumFreezingUrl },
+}) => {
+  const Link = useSmartAnchor();
+  return (
+    <div>
+      <h4>Impact</h4>
+      <p>
+        Transaction ID space utilization is high and approaching to wraparound.
+      </p>
+      <p>
+        Postgres runs autovacuums regularly and this helps keeping transaction
+        ID space utilization low by freezing old transaction IDs. There is also
+        the anti-wraparound autovacuum specific for freezing, when the regular
+        autovacuums are either not freezing or simply not running. This
+        anti-wraparound autovacuum will be triggered when utilization (age)
+        exceeds the threshold, specified in{" "}
+        <code>autovacuum_freeze_max_age</code>.
+      </p>
+      <p>
+        In order to avoid wraparound failure, old transaction IDs must be
+        freezed by VACUUMs and this is not avoidable. The more old transaction
```

Comment:
```suggestion
        frozen by VACUUMs. The more old transaction
```

Reply:
Since the rest here is very similar to the above, I think the above comments apply.